tests/wicked/before_test.pm

```diff
@@ -44,37 +44,18 @@ sub run {
     $self->download_data_dir();
 
     if (check_var('IS_WICKED_REF', '1')) {
+        # Common REF Configuration
         record_info('INFO', 'Setup DHCP server');
-        zypper_call('--quiet in dhcp-server', timeout => 200);
+        zypper_call('--quiet in dhcp-server openvpn', timeout => 200);
         $self->get_from_data('wicked/dhcp/dhcpd.conf', '/etc/dhcpd.conf');
-        if (check_var('WICKED', 'basic') || check_var('WICKED', '2nics')) {
-            assert_script_run(sprintf("ip a a %s dev %s", $self->get_ip(type => 'second_card'), $ctx->iface()));
-        }
         file_content_replace('/etc/sysconfig/dhcpd', '--sed-modifier' => 'g', '^DHCPD_INTERFACE=.*' => 'DHCPD_INTERFACE="' . $ctx->iface() . '"');
         systemctl 'enable dhcpd.service';
         systemctl 'start dhcpd.service';
-    }
-    if (check_var('WICKED', 'basic')) {
-        assert_script_run("rcwickedd restart");
-        unless (get_var('IS_WICKED_REF')) {
-            # Remove previous static config and leave dynamic one, only for SUT
-            assert_script_run('rm /etc/sysconfig/network/ifcfg-' . $ctx->iface());
-            $self->get_from_data("wicked/dynamic_address/ifcfg-eth0", '/etc/sysconfig/network/ifcfg-' . $ctx->iface());
-        }
-    }
-    elsif (check_var('WICKED', 'advanced') || check_var('WICKED', 'startandstop')) {
-        assert_script_run("rcwickedd restart");
-        record_info('INFO', 'Setup OpenVPN');
-        zypper_call('--quiet in openvpn', timeout => 200);
-        if (check_var('WICKED', 'advanced')) {
-            record_info('INFO', 'Setup OVS');
-            zypper_call('--quiet in openvswitch libteam-tools libteamdctl0 python-libteam', timeout => 200);
-            assert_script_run("systemctl start openvswitch");
-        }
-    }
-    elsif (check_var('WICKED', '2nics')) {
-        assert_script_run('rm /etc/sysconfig/network/ifcfg-' . $ctx->iface());
-        assert_script_run("rcwickedd restart");
+    } else {
+        # Common SUT Configuration
```

Comment:
what about such version : 
```
my $package_list='openvswitch openvpn';
$package_list .= 'libteam-tools libteamdctl0 python-libteam gcc' if check_var('WICKED', 'advanced');
zypper_call('-q in ' . $package_list, timeout => 400);
```
also if you double check code which you deleted we were giving 200 second to install of openvpn and 200  for all other thing. I would go for just sum them

Reply:
ok, looks better